lib/select-builder.js

```diff
@@ -92,232 +77,162 @@ class SelectBuilder extends QueryBuilder {
   }
 
   distinct() {
-    this.operations.set('selectDistinct', true);
+    this.operations.distinct = true;
     return this;
   }
 
-  where(key, cond, value) {
-    cond = parseCondition(cond.toUpperCase());
-    this.operations.get('where').push({
-      key: this.escapeKey(key),
-      value,
-      cond,
-    });
+  where(field, operator, value, not) {
```

Comment:
This would better be left as is and added a new (like `_where` function) than to change public API for this. Furthermore, it will probably be extended later to support more use cases and is even worse for public API.

Reply:
It's v0.1.0 now, adding argument will not break compatibility but I have multiple proposals how to change public API to brake it )))

Reply:
Sure but since this is a 'refactroing' the API should stay the same and breaking changes are to be discussed in separate PRs.